test/models/comment_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
I think unit tests need to be added before we can consider merging this PR. It is important that we don't introduce breaking changes and have to another big rollback.